chains/solana/contracts/programs/execution-buffer/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "execution_buffer"
+version = "0.1.0-dev"
+description = "Created with Anchor"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "lib"]
+name = "execution_buffer"
+
+[features]
+no-entrypoint = []
+no-idl = []
+no-log-ix-name = []
+cpi = ["no-entrypoint"]
+default = []
+
+[dependencies]
+solana-program = "1.17.25" # pin solana to 1.17
+anchor-lang = { version = "0.29.0", features = ["init-if-needed"] }
+anchor-spl = "0.29.0"
+bytemuck = "1.7"
+ethnum = "1.5"
```

Comment:
Do we really need ethnum and anchor-spl?

Reply:
Nope, just a leftover. Removing it.